aws/aws_s3_bucket/s3_static_website/main.tf

```diff
@@ -0,0 +1,61 @@
+# Summary: Setup a static website inside the S3 Bucket 
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 1.0.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.38"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+provider "aws" {
+  region = "us-east-1"
+  default_tags {
+    tags = {
+      cs_terraform_examples = "aws_s3_bucket/s3_static_website"
+    }
+  }
+}
+
+
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
```

Comment:
Change Documentation link to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#static-website-hosting